src/test/java/it/eng/idsa/businesslogic/processor/sender/SenderParseReceivedDataProcessorBodyBinaryTest.java

```diff
@@ -38,6 +39,9 @@
 	@Mock
 	private RejectionMessageService rejectionMessageService;
 
+	@Value("${application.isEnabledDapsInteraction}")
```

Comment:
Why using this @Value spring annotation in junit tests?
Will not work here because SPring context is not being loaded